### PR TITLE
Support ScyllaDB YCSB workload

### PR DESCRIPTION
**Pull request by sitano**
This PR adds support to a YCSB workload A scenario for Scylla.

It also extends the functionality to support:

- multi statements keyspace preparation
- log collection for ycsb

## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

File: test-cases/longevity/longevity-ycsb-a-10M.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+n_db_nodes: 3
+n_loaders: 2
+n_monitor_nodes: 1
+
+instance_type_db: 'i3.4xlarge'
+instance_type_loader: 'm5.2xlarge' # 8 vCPU (4 cores), 32 GB RAM
+
+nemesis_class_name: 'NoOpMonkey'
+nemesis_interval: 5
+ssh_transport: 'libssh2'
```
Comment (fruch) [inline test-cases/longevity/longevity-ycsb-a-10M.yaml:47]: any reason it's here ? and not using the default ? (I don't remember which one is the default...)
if there is please add a comment about it

File: test-cases/longevity/longevity-ycsb-a-10M.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+authenticator: 'PasswordAuthenticator'
+authenticator_user: cassandra
+authenticator_password: cassandra
+authorizer: 'CassandraAuthorizer'
+
+user_prefix: 'ivan-ycsb-a'
```
Comment (fruch) [inline test-cases/longevity/longevity-ycsb-a-10M.yaml:54]: I think you can take you name out of it :)

**Comment by fruch** [comment]
@sitano did you run those locally ? or in Jenkins using BYO job ?
would be nice to see the outcome of such a run

@roydahan  you think we might want to use those regularly? , if so, it would be nice to have pipelines for them too...

**Review by fruch — APPROVED** [review]
LGTM, 

few comments about the some of the yaml parameters,
and if we want pipelines/jobs for those

File: test-cases/longevity/longevity-ycsb-a-10M.yaml (hunk 1 above)
Comment (sitano) [reply test-cases/longevity/longevity-ycsb-a-10M.yaml:47]: no. blindly copying it from somewhere. remove?

**Comment by fruch** [comment]
@sitano I would also love to hear a bit about the context of why you want those for...

File: test-cases/longevity/longevity-ycsb-a-10M.yaml (hunk 1 above)
Comment (fruch) [reply test-cases/longevity/longevity-ycsb-a-10M.yaml:47]: yes, please do

**Comment by sitano** [comment]
@fruch I am currently testing CRDB vs ScyllaDB with YCSB workloads. So I needed some support for it from SCT. I ended up have those changes running with Jenkins BYO https://jenkins.scylladb.com/view/staging/job/scylla-staging/job/ivan/job/ivan-byo-longevity/23/.

I don't think it makes sense to run it regularly but I thought that maybe those changes would be useful for anyone in the future.

File: test-cases/longevity/longevity-ycsb-a-10M.yaml [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+# 10M / 44K = ~3 minutes
+#
+# 240 = 4 hours * 60
+# 480 = 4 hours * 60
+# 10900 = 7 days * 24 hours a day * 60 minutes an hour
+test_duration: 480
+
+pre_create_keyspace: [
+  "CREATE KEYSPACE ycsb WITH REPLICATION = {'class' : 'SimpleStrategy', 'replication_factor': 3 };",
+  "CREATE TABLE ycsb.usertable (y_id varchar primary key, field0 varchar, field1 varchar, field2 varchar, field3 varchar, field4 varchar, field5 varchar, field6 varchar, field7 varchar, field8 varchar, field9 varchar);"
```
Comment (fruch) [inline test-cases/longevity/longevity-ycsb-a-10M.yaml:11]: BTW, I have code doing this somewhere in SCT.

But I like this approach even better.

File: test-cases/longevity/longevity-ycsb-a-10M.yaml (hunk 1 above)
Comment (sitano) [reply test-cases/longevity/longevity-ycsb-a-10M.yaml]: ok. removed libssh2.

File: test-cases/longevity/longevity-ycsb-a-10M.yaml (hunk 2 above)
Comment (sitano) [reply test-cases/longevity/longevity-ycsb-a-10M.yaml]: fixed prefixes

**Comment by sitano** [comment]
btw, I am not sure about log collecting code. But I currently don't have time to invest more into it. Current run in Jenkins will took a day so I will see)

**Comment by sitano** [comment]
btw, maybe you would like to have this in debug level:

> self.log.info('pre_create_keyspace cmd: {}'.format(cmd))

**Comment by fruch** [comment]
> @fruch I am currently testing CRDB vs ScyllaDB with YCSB workloads. So I needed some support for it from SCT. I ended up having those changes running with Jenkins BYO https://jenkins.scylladb.com/view/staging/job/scylla-staging/job/ivan/job/ivan-byo-longevity/23/.
> 
> I don't think it makes sense to run it regularly but I thought that maybe those changes would be useful for anyone in the future.

If that's the case, if you can drop this a comment on the top of those yamls, it would be nice so we won't need to guess why are there here and for what.

**Comment by fruch** [comment]
> btw, maybe you would like to have this in debug level:
> 
> > self.log.info('pre_create_keyspace cmd: {}'.format(cmd))

yeah you got a point there, I missed that while reading....

**Comment by sitano** [comment]
Done

**Review by fruch — APPROVED** [review]
LGTM

**Pull request merged**